apps/server/src/modules/user/service/user.service.ts

```diff
@@ -124,8 +128,11 @@ export class UserService {
 	}
 
 	async deleteUser(userId: EntityId): Promise<number> {
+		this.logger.log({ action: 'Deleting user ', userId });
 		const deletedUserNumber: Promise<number> = this.userRepo.deleteUser(userId);
 
+		this.logger.log({ action: 'Deleted user ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of deleted userId. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114